packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/InternalCustomTabsActivitiesHelper.java

```diff
@@ -0,0 +1,127 @@
+package expo.modules.webbrowser;
```

Comment:
As we're aiming for Kotlin supremacy, I'll convert this to kotlin 😉

Reply:
Let me do that in another PR, will you? This is actually not a new file, just moving most of other file here. And I don't want (following your suggestion) to do two things in one PR.